inst/js/trajectory_alignment.js

```diff
@@ -390,6 +390,42 @@ class trajectory_alignment {
           .range(["black", "#9cab7c", "#c3d69b", "#FFFFCC", "#fde291", "#e07b73"])
           legend_order = ["worse", "STEPS", "APS", "SDS", "NZE_2050"];
         ;
+      } else if (selected_source == "GECO2022") {
+        color = d3.scaleOrdinal()
+          .domain(["production", "1.5C", "NDC_LTS", "Reference", "worse"])
+          .range(["black", "#9cab7c", "#c3d69b", "#FFFFCC", "#e07b73"])
+          legend_order = ["worse", "Reference", "NDC_LTS", "1.5C"];
+        ;
+      } else if (selected_source == "ISF2021") {
+        color = d3.scaleOrdinal()
+          .domain(["production", "NZE", "worse"])
+          .range(["black", "#9cab7c", "#e07b73"])
+          legend_order = ["worse", "NZE"];
+        ;
+      } else if (selected_source == "WEO2022") {
+        color = d3.scaleOrdinal()
+          .domain(["production", "NZE_2050", "APS", "STEPS", "worse"])
+          .range(["black", "#9cab7c", "#FFFFCC", "#fde291", "#e07b73"])
+          legend_order = ["worse", "STEPS", "APS", "NZE_2050"];
+        ;
+      } else if (selected_source == "GECO2023") {
+        color = d3.scaleOrdinal()
+          .domain(["production", "1.5C", "NDC-LTS", "Reference", "worse"])
+          .range(["black", "#9cab7c", "#c3d69b", "#FFFFCC", "#e07b73"])
+          legend_order = ["worse", "Reference", "NDC-LTS", "1.5C"];
+        ;
+      } else if (selected_source == "ISF2023") {
+        color = d3.scaleOrdinal()
+          .domain(["production", "1.5\xc2\xb0C", "worse"])
+          .range(["black", "#9cab7c", "#e07b73"])
+          legend_order = ["worse", "1.5\xc2\xb0C"];
+        ;
+      } else if (selected_source == "WEO2023") {
+        color = d3.scaleOrdinal()
+          .domain(["production", "NZE_2050", "APS", "STEPS", "worse"])
+          .range(["black", "#9cab7c", "#FFFFCC", "#fde291", "#e07b73"])
+          legend_order = ["worse", "STEPS", "APS", "NZE_2050"];
+        ;
```

Comment:
The WEO 2023 scenario seems to be called `NZE` here (in contrast to WEO 2022 where it is`NZE_2050`):
``` r
library(pacta.scenario.preparation)

weo_2023 |> 
  dplyr::distinct(scenario)
#> # A tibble: 3 × 1
#>   scenario
#>   <chr>   
#> 1 STEPS   
#> 2 APS     
#> 3 NZE

weo_2022 |> 
  dplyr::distinct(scenario)
#> # A tibble: 3 × 1
#>   scenario
#>   <chr>   
#> 1 APS     
#> 2 NZE_2050
#> 3 STEPS
```

<sup>Created on 2024-03-25 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
I thought that `WEO2023` would have `NZE_2050` after https://github.com/RMI-PACTA/pacta.scenario.preparation/pull/140 no?

Reply:
Oh bizarre, maybe I'm using an old version installed locally

Reply:
related https://github.com/RMI-PACTA/workflow.transition.monitor/pull/288